storybookjs/storybook
# UI: Fix `showPanel=false` doesn't work on mobile view

Issue: https://github.com/storybookjs/storybook/issues/18386

## What I did

- Support `options.showPanel` in `code/ui/manager/src/components/layout/mobile.tsx`

## How to test

Run a sandbox in development mode with new `manager.js`

```js
// sandbox/react-vite-default-ts/.storybook/manager.js
import { addons } from '@storybook/addons';

addons.setConfig({ showPanel: false });
```

Switch to mobile view and verify the `Addons` tabs is removed from bottom tabs bar
